### ISSUE TITLE
Keeping the aspect ratio when using the ijkplayer

### ISSUE THREAD

**Issue by animaonline** [Manuiq/ZoomableTextureView#7]
Any ideas on how to achieve that?

**Comment by Manuiq** [comment]
Yes, in it's current form ZoomableTextureView doesnt preserve aspect ratio but you are free to resize at as any regular View as soon as you receive the video metadata from the ijkplayer
